src/fides/api/schemas/connection_configuration/connection_secrets_base_aws.py

```diff
@@ -14,6 +14,7 @@ class BaseAWSSchema(ConnectionConfigSecretsSchema):
     auth_method: AWSAuthMethod = Field(
         title="Authentication Method",
         description="Determines which type of authentication method to use for connecting to Amazon Web Services.",
+        default=AWSAuthMethod.SECRET_KEYS,
```

Comment:
i think my only question around this is whether it plays nicely with the UI? namely, is the field populated with this value?

Reply:
great!